src/components/RegistrationForm/Form/FieldValidation/FieldValidation.styled.js

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
+
+export const TextValidation = styled.p`
+  font-family: Gilroy-Regular, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.5px;
+
```

Comment:
and there is no reason to make an empty line..

Reply:
This is a draft version of the code.